assets/app/view/auction_round.rb

```diff
@@ -57,7 +57,7 @@ def render_input
         value: @round.min_bid(@selected_company),
         step: step,
         min: @selected_company.min_bid + step,
-        max: @current_entity.cash,
+        max: @round.max_bid(@current_entity, @selected_company),
```

Comment:
looks like this a UI only fix?

that's fine for me but making sure that's your intent

Reply:
My impression is that we often enforce rules only at the UI level (e.g., not giving you a pass button if you need to sell shares), so it seemed okay to me.

Reply:
ideally it would be in both places, but sometimes i'm afraid to do it in the server because it will break too many games... but the future is here now